components/TagPage.vue

```diff
@@ -154,8 +154,8 @@
               :image="getArticleImage(story.leadAsset, story.ancestry[0].slug, story.listingImage)"
               :image-height="560"
               :image-width="413"
-              :image-max-height="560"
-              :image-max-width="413"
+              :image-max-height="getArticleImageHeight(story.leadAsset, story.listingImage) || 560"
+              :image-max-width="getArticleImageWidth(story.leadAsset, story.listingImage) || 413"
```

Comment:
In the case that we don't know the image dimensions, we can fall back to `Infinity`, i.e. no maximum height.
(The simple responsive image component actually does this internally when this parameter isn't set)
Even if the 3x image size request dimensions are larger than the original image dimensions, the backend will still return the biggest image it's able to.
Our 2x or 3x sizes might not be quite "2 x" or "3 x" in that case but will still be better than the fixed size for displays that need it.

for example:
```:image-max-height="getArticleImageHeight(story.leadAsset, story.listingImage) || Infinity"
:image-max-width="getArticleImageWidth(story.leadAsset, story.listingImage) || Infinity"```

(There's a few other places in the code like this)